agent/functional_tests/tests/functionaltests_test.go

```diff
@@ -653,6 +653,72 @@ func telemetryTestWithStatsPolling(t *testing.T, taskDefinition string) {
 	require.NoError(t, err, "Waiting for task stop failed")
 }
 
+// for now this is just used to start and run the storageStats task
+func telemetryStorageStatsTest(t *testing.T, taskDefinition string) {
```

Comment:
can you skip the test when the instance has less than 2gb memory? example https://github.com/aws/amazon-ecs-agent/blob/master/agent/functional_tests/tests/functionaltests_test.go#L472-L473. Otherwise the test will fail on small instance.

Reply:
sure.  I'll add that.